ProjFS.Mac/PrjFSKextTests/TestLocks.cpp

```diff
@@ -0,0 +1,37 @@
+#include "../PrjFSKext/Locks.hpp"
+
```

Comment:
In this file, I did the minimum required to get the tests to build & run. We could of course get fancy.

Reply:
I think this is an exellent first step and I wouldn't get fancy yet :-)